pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -69,8 +69,8 @@ const (
 	ErrReasonNodeConflict ConflictReason = "node(s) had volume node affinity conflict"
 	// ErrReasonNotEnoughSpace is used when a pod cannot start on a node because not enough storage space is available.
 	ErrReasonNotEnoughSpace = "node(s) did not have enough free storage"
-	// ErrReasonPVNotExist is used when a PVC can't find the bound persistent volumes"
-	ErrReasonPVNotExist = "pvc(s) bound to non-existent pv(s)"
+	// ErrReasonPVNotExist is used when a pod has PVC(s) but can't find the bound persistent volume(s)"
+	ErrReasonPVNotExist = "node(s) due to pvc(s) bound to non-existent pv(s)"
```

Comment:
isn't this a pod level error, not a node level one? like all nodes would not be eligible if the pod's PVC was bound to non-existent PV.

Reply:
right but [FindPodVolumes](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/framework/plugins/volumebinding/binder.go#L133) is invoked for all nodes and the number of not available ones would be accumulated because everyone node would not be eligible since the PV is not found.

This change just cleans up the message to indicate that the node is not selected due to this reason.
The previous message of something like:

```
0/2 nodes are available: 2 pvc(s) bound to non-existent pv(s)
```
can be confusing since its not 2 PVCs bound to non-existent PVs, but rather 2 nodes not available to be scheduled due to pod's PVC(s) bound to non-existent PV(s).